apps/voting/app/src/hooks/useExtendedVoteData.js

```diff
@@ -1,4 +1,6 @@
 import { useMemo } from 'react'
+import BN from 'bn.js'
+import { formatTokenAmount } from '@aragon/ui'
 import { useAragonApi } from '@aragon/api-react'
```

Comment:
Now that I'm seeing the order, shouldn't we move `@aragon/ui` below `@aragon/api-react`?